api/api.yaml

```diff
@@ -1619,6 +1615,44 @@ paths:
                       items:
                         type: string
                       description: The scopes to use for the connection (optional).
+                oauth2AuthorizationCode:
+                  type: object
+                  properties:
+                    accessToken:
```

Comment:
inside the `accessToken` object, `token` should be required (but `accessToken` itself is not required)